classes/simplemap.php

```diff
@@ -1567,13 +1560,13 @@ function createSidebarEntry(marker, locationData, searchData) {
 				// Phone, email, and fax numbers
 				html += '<div class="result_phone">';
 				if (locationData.phone != null && locationData.phone != '') {
-					html += phone_text + ': ' + locationData.phone;
+					html += '<span class="result_phone">' + phone_text + ': <a href="tel:' + locationData.phone + '">' + locationData.phone + '</a></span>';
```

Comment:
__[code_sniffer]__
* Line exceeds 120 characters; contains 156 characters
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->